assets/scripts/neighbors_db.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE IF NOT EXISTS users (
     State VARCHAR(100) NULL,
     Street VARCHAR(100) NULL,
     UserType TINYINT NOT NULL DEFAULT 1,
+    CONSTRAINT idx_users_email UNIQUE (Email),
```

Comment:
Do you have a migration script that does this sort of thing?

Reply:
Nope, mainly because I was using Heroku previously, which doesn't lend itself well to that on the "I'm a cheapskate" plan. But I'll probably need to build something when I switch over to Digital Ocean.